easybuild/easyblocks/b/bowtie2.py

```diff
@@ -16,12 +16,13 @@
 @author: Fotis Georgatos (Uni.Lu)
 @author: Kenneth Hoste (Ghent University)
 """
-
+from distutils.version import LooseVersion
 import os
 import shutil
 
 from easybuild.easyblocks.generic.configuremake import ConfigureMake
 from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.filetools import copy_file
 
 
 class EB_Bowtie2(ConfigureMake):
```

Comment:
`MakeCp` implements a generic procedure where files specified in `files_to_copy` are being copied

Here we implement a custom `install_step` for Bowtie2 that copies very specific files/directories, so whether we derive from `ConfigureMake` or `MakeCp` doesn't make a difference, except that if we derive from `MakeCp` we have to take extra action to avoid that `files_to_copy` *has* to be specified (while it doesn't make sense to specify it in this specific case).